cmd/kubelet/app/server.go

```diff
@@ -175,6 +181,12 @@ func (s *KubeletServer) Run(_ []string) error {
 		HighThresholdPercent: s.ImageGCHighThresholdPercent,
 		LowThresholdPercent:  s.ImageGCLowThresholdPercent,
 	}
+
+	cloud := cloudprovider.InitCloudProvider(s.CloudProvider, s.CloudConfigFile)
+	if cloud != nil {
+		glog.Infof("Successfully initialized cloud provider: %s from the config file: %s\n", s.CloudProvider, s.CloudConfigFile)
+	}
```

Comment:
Any reason not to return err on failure? We should glog.Errorf() if we can't return error